src/dftbp/mixer/andersonmixer.F90

```diff
@@ -73,19 +73,37 @@ module dftbp_mixer_andersonmixer
 
     !> Stored prev. charge differences
     ${TYPE}$(dp), allocatable :: prevQDiff(:,:)
+      contains
+        procedure :: init => TAndersonMixer${LABEL}$_initFromStruct
```

Comment:
Usually, I prefer to have the "constructor" not as a type bound proceduref (as an eventual derived class won't be able to change the signature of the constructor). We should probably make it a proper constructor giving back `type(TAndersonMixer)` instead of `class(TAndersonMixer)`.